source/product-stories/chargy.html.erb

```diff
@@ -0,0 +1,154 @@
+  image: "/product-stories/chargy/chargy-prototype.jpg"
+  image_description: "A prototype of char.gy"
+
+in_numbers_2:
+  title: "char.gy in numbers"
+  project: "chargy"
+  statistics:
+    - number: "7.36kW"
+      text: "Power levels provided up to"
+    - number: "5"
+      text: "Live chargepoints now available"
+
+software_platform:
+  title: "A software platform for manage EV charging and payments"
+  content: "Through user-led, agile approaches, the software platform was iteratively built around the physical step-by-step charging process, from a driver locating an available chargepoint through a map, to commencing a charging session and making payments through a range of tariffs. The char.gy application is now compatible with Apple Pay, allowing drivers to make single-touch purchases on the ‘Pay As You Go’ option for quicker and easier ad-hoc charging."
+  image: "/product-stories/chargy/chargy-phones.png"
+  image_description: "char.gy app on phone"
+
+public_lampost_chargepoint:
+  title: "The UK’s first open, public lamppost EV chargepoint"
+  content: "The launched software application allows for drivers to manage their end-to-end charging journey from their smartphone (from locating an available chargepoint to processing payments). Passing all relevant UK hardware and electrical legislation, including ELEXON certification, char.gy has been installed onto public streets across London and Buckinghamshire, with further units due to be installed in 2019 to bring open, public lamppost EV charging to more residents."
+  image: "/product-stories/chargy/car-on-street.jpg"
+  image_description: "Car on street charging with char.gy"
+
+quote_chris_2:
+  quote: "The Unboxed team has been core to the success of getting char.gy onto the streets. Our next goal is to bring open electric vehicle charging to more residents across the UK — something we are working with Unboxed to do across 2019."
+  source: "Chris McDowell, Chief Product Officer — char.gy"
+
+quote_henry:
+  quote: "The char.gy application has been designed and developed to meet a number of different elements — from communication with the charge points, enabling drivers to find available chargepoints through map integrations, reliability and performance monitoring, to beginning a charge through the front-end and handling payments via the most convenient methods. Our user-led approach has enabled the team to iteratively build a web platform to reach as many people as possible that is aligned with user needs."
+  source: "Henry Turner, Senior Developer - Unboxed"
```

Comment:
Maybe worth checking with Henry about how we want to list his role under his quote - I had an idea that he's CTO of Char.gy - if that's right then maybe we want to use that here?

Reply:
I've asked Henry, he says it's better that he stays as senior developer because it's a story associated with Unboxed

Reply:
Good point!